README.md

```diff
@@ -156,8 +156,18 @@ $mail->setFrom('from@address.com', 'Support');
 $mail->setSubject('Hello, thats mail with attachments!!');
 $mail->setHtml('<html><body><strong>Attachments</strong> in mail</body></html>');
 $mail->addRecipientTo('recipient email address');
-$mail->addAttachment('path to local file 1');
-$mail->addAttachment('path to local file 2');
+$mail->addAttachment(
+            new LocalFileAttachment(
+                'path to local file 1',
+                 'optional file name'
+            )
+        );
+$mail->addAttachment(
+            new Base64Attachment(
+                'file name',
+                base64 file content
```

Comment:
Tutaj zmieńmy, żeby był jakiśprzylładowy base, tak żeby dało sie ten kod zrobić ctrl+c i ctrl+v i żeby działało, bo tak to jest niepoprawny php. Jeszcze przestrzenie nazw dla LocalFileAttachment i Base64Atachment brakuje w `use`

Reply:
base'a nie bede dawal bo to zaciemni przyklad, dodalem drugi parametr w '' zeby byl to poprawny php. Na tej samej zasadzie bylo z LocalFileAttachment. Dodalem brakujace use